docs/web/authentication.md

```diff
@@ -362,6 +363,9 @@ Is it possible to generate a token from command line which can be used to
 authenticate in the name of the given user. This way no need to store passwords
 in text files. For more information [see](#personal-access-token).
 
+Note: do not give protocols in the credentials section. Use `my.company.org:443`
+instead of `https://my.company.org:443`.
```

Comment:
what happens if I give a protocol? It is not really intuitive that the protocol part of a standard URL format is prohibited. Couldn't we cut it if it isn't necessary?

Reply:
@gyorb already refactored this in #1460 pull request but it is still in work in progress status.

Reply:
@bruntib I refactored this and I now cut the protocols and product names from the credentials.